src/lib/viewers/controls/annotations/__tests__/AnnotationsTargetedTooltip-test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { ReactWrapper, mount } from 'enzyme';
+import AnnotationsTargetedTooltip from '../AnnotationsTargetedTooltip';
+
+const mockUseContext = jest.fn().mockImplementation(() => ({
+    experiences: {
+        tooltipFlowAnnotationsExperience: {
+            canShow: true,
+            onClose: jest.fn(),
+            onComplete: jest.fn(),
+            onShow: jest.fn(),
+        },
+    },
+    setIsForced: jest.fn(),
+}));
+
+React.useContext = mockUseContext;
```

Comment:
Can we use `jest.spyOn(React, 'useContext')` in a `beforeEach`?

Reply:
The problem when using `jest.spyOn(React, 'useContext')` versus how I have it, is with the spy version that experiences value is `{}` which results in shouldTarget always being false.